src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -481,9 +483,9 @@ export default class FilingHistoryList extends Vue {
     // Also update pending COA data.
     let isCoaPending = false
     let coaEffectiveDate: Date = null
-    const blockerFiling = this.historyItems.find(item => {
+    const blockerFiling = this.historyItems.some(item => {
```

Comment:
`some` may or may not be faster than `find`, but it returns a boolean so the `!!` on line 498 is no longer needed.